src/app/controllers/login.js

```diff
@@ -43,9 +46,8 @@ angular.module('flightNodeDemo')
 
 							// response.data has the the access_token and expires_in (seconds).
 							// Need to record the actual expiration timestamp, not just the duration.
-							var expiresAt = moment().add(response.data.expires_in, 's');
-
-							localStorage.setItem('jwt', JSON.stringify({ expiresAt: expiresAt, access_token: response.data.access_token }));
+							var expiresAt = moment().add(response.data.expires_in, 's').toDate();
+							oauthRequest.setToken(response.data.access_token, expiresAt);
```

Comment:
The new `oauthRequest` service puts the JWT into a cookie instead of using localStorage, as [recommended by StormPath](https://stormpath.com/blog/where-to-store-your-jwts-cookies-vs-html5-web-storage/). However, this is just a step in the right direction on security, since I haven't implemented any CSRF token strategy yet.